packages/effect/src/Arbitrary.ts

```diff
@@ -0,0 +1,580 @@
+/**
+ * @since 3.10.0
+ */
+
+import * as Arr from "./Array.js"
+import * as FastCheck from "./FastCheck.js"
+import * as errors_ from "./internal/schema/errors.js"
+import * as filters_ from "./internal/schema/filters.js"
+import * as util_ from "./internal/schema/util.js"
+import * as Option from "./Option.js"
+import * as Predicate from "./Predicate.js"
+import type * as Schema from "./Schema.js"
+import * as AST from "./SchemaAST.js"
+
+/**
+ * @category model
+ * @since 3.10.0
+ */
+export interface LazyArbitrary<A> {
+  (fc: typeof FastCheck): FastCheck.Arbitrary<A>
+}
+
+/**
+ * @category hooks
+ * @since 3.10.0
+ */
+export const ArbitraryHookId: unique symbol = Symbol.for("effect/Schema/ArbitraryHookId")
```

Comment:
```suggestion
export const ArbitraryHookId: unique symbol = Symbol.for("effect/Arbitrary/ArbitraryHookId")
```
I think this makes more sense from file structure.

Reply:
There are a few others in the other modules too.